src/app/page.tsx

```diff
@@ -0,0 +1,9 @@
+import { LinkGenerator } from "@/components/LinkGenerator";
+
+export default async function Home() {
+  return (
+    <main>
+      <LinkGenerator />
+    </main>
```

Comment:
nit: I would consider moving content of the `LinkGenerator`component into here until we know it needs to be a separate component. The level of indirection seems not needed here at the this moment in time.